src/api-handler.ts

```diff
@@ -30,18 +36,28 @@ class APIRequest {
    *                     application/x-www-form-urlencoded data in order for it
    *                     to be parsed properly.
    */
-  constructor(params) {
+  constructor(params: RequestParams) {
     this.method = params.method;
     this.path = params.path;
     this.query = params.query || {};
     this.body = params.body || {};
   }
 }
 
+interface RequestParams {
+  method: string;
+  path: string;
+  query: {};
+  body: {};
+}
+
 /**
  * Convenience class to build an API response.
  */
-class APIResponse {
+export class APIResponse {
+  public status: number;
+  public contentType?: string;
+  public content?: string;
```

Comment:
I think it would be more convenient for the user if this would be of type `any`.
I used `string` here because of the docs.

Reply:
My only hesitation is that if the user passes in an object, should we automatically convert to JSON? If you just do `${contentType}`, you'll end up getting `"[object Object]"`.

Reply:
What about an assertion?
The `Property` constructor does this for the `propertyDescr`.
https://github.com/mozilla-iot/gateway-addon-node/blob/76a2186831d8a70c0512400e0eae30759ed2b598/lib/property.js#L55

Reply:
Fair point.
Then how about leaving the type unchanged and adding an assertion like in the `Property` constructor (https://github.com/mozilla-iot/gateway-addon-node/blob/76a2186831d8a70c0512400e0eae30759ed2b598/lib/property.js#L55) to ensure that the user passes a string?
This ensures that there is no unexpected serialization.

Reply:
That works for me.